readme.md

```diff
@@ -504,6 +504,7 @@
 - [Theoretical Computer Science](https://github.com/mostafatouny/awesome-theoretical-computer-science#readme) - The interplay of computer science and pure mathematics, distinguished by its emphasis on mathematical rigour and technique.
 - [Conversational AI](https://github.com/jyguyomarch/awesome-conversational-ai#readme) - Build awesome chatbots and digital assistants.
 - [Generative AI](https://github.com/steven2358/awesome-generative-ai#readme) - Automatically generate a wide range of unique content in text, image and audio format.
+- [Computational Intelligence in Sports](https://github.com/firefly-cpp/awesome-computational-intelligence-in-sports#readme) - Research, methods, datasets, and software related to use of Computational Intelligence in Sports.
```

Comment:
> Your entry here should include a short description about the project/theme of the list. **It should not describe the list itself.**

Reply:
I am sorry for the a bit longer reply since I was on holidays. I have fixed the description according to your instructions in [this commit](https://github.com/sindresorhus/awesome/pull/2565/commits/4bb471e2cf0e48f602cf53db1d3ba89b52902eac). I believe the repo is now fixed, but I believe the check will still fail because the Linder does _not like_ emojis, although the ToC works correctly.